src/Components/News/NewsCreateEdit.js

```diff
@@ -0,0 +1,231 @@
+}
+//dropzone--------------------------------
+const thumbsContainer = {
+    display: 'flex',
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    marginTop: 16
+  };
+  
+  const thumb = {
+    display: 'inline-flex',
+    borderRadius: 2,
+    border: '1px solid #eaeaea',
+    marginBottom: 8,
+    marginRight: 8,
+    width: 100,
+    height: 100,
+    padding: 4,
+    boxSizing: 'border-box'
+  };
+  
+  const thumbInner = {
+    display: 'flex',
+    minWidth: 0,
+    overflow: 'hidden'
+  };
+  
+  const img = {
+    display: 'block',
+    width: 'auto',
+    height: '100%'
+  };
+
+function NewsCreateEdit() {
+    
+    const { id } = useParams() //Esto valida si se esta editando o creando un post mediante paso de parametros
+    //const refP = useRef() //cambia la visibilidad del <p> "insertar imagen"
+    const urlCategoria = 'http://ongapi.alkemy.org/public/api/categories';
+    const urlEditarNovedad = `http://ongapi.alkemy.org/public/api/news/${id}`
+    const urlCrearNovedad = 'http://ongapi.alkemy.org/public/api/news'
+    const [categorias, setcategorias] = useState([]) //obtiene las categorias del endpoint
+    const [initialValues, setInitialValues] = useState(datosForm);
+    const [files, setFiles] = useState([]);
+    const {getRootProps, getInputProps} = useDropzone({
+      accept: 'image/png, image/jpg',
+      onDrop: acceptedFiles => {
+        setInitialValues({...initialValues, image: (acceptedFiles.map(file => Object.assign(file, {
+            preview: URL.createObjectURL(file)
+        })))})
+        setFiles(acceptedFiles.map(file => Object.assign(file, {
+          preview: URL.createObjectURL(file)
+        })));
+      }
+    });
+    
+    const thumbs = files.map(file => (
+      <div style={thumb} key={file.name}>
+        <div style={thumbInner}>
+          <img
+            src={file.preview}
+            style={img}
+            alt='img not found'
+          />
+        </div>
+      </div>
+    ));
+  
+    useEffect(() => () => {
+      // Make sure to revoke the data uris to avoid memory leaks
+      files.forEach(file => URL.revokeObjectURL(file.preview));
+    }, [files]);
+
+    const handleChange = (e) => { //actualiza el estado de title y category
+        if (e.target.name === 'title') {
+            setInitialValues({ ...initialValues, title: e.target.value })
+        } if (e.target.name === 'category') {
+            setInitialValues({ ...initialValues, category: e.target.value })
+        }
+    }
+
+   
+    const contentChange = (e, editor) => { //actualiza el estado de content
+        const data = editor.getData()
+        setInitialValues({
+            ...initialValues, content: data
+        })
+    }
+
+    const handleSubmit = (e) => { //envia los datos dependiendo del id hace una peticion post o put
+        const datosConEtiquetas = initialValues.content
+        const datosSinEtiquetas = datosConEtiquetas.replace(/<[^>]+>/g, ''); //convierte a texto plano el content
```

Comment:
please use english names

Reply:
create a function in utils to convertCkEditorText:
```
const convertCkEditorText = (text) => text.replace(/<[^>]+>/g, ''); 
```
then, create here the object that will be sent to the service with the result of that function
```
const newsToSubmit = { ...initialValues, content: convertCkEditorText(initialValues.content) };
```

Reply:
remember that initialValues should not exists and the name of that object should be news